src/Data/MongoDBMock/participants.js

```diff
@@ -0,0 +1,127 @@
+const participants = [
+  {
+    participantId: 'matylda1234',
+    email: 'matylda@op.pl',
+    password: 'dupabiskupa',
```

Comment:
password will be hashed but that is when the backend comes in

Reply:
true!